distribution/tests/pom.xml

```diff
@@ -0,0 +1,220 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.elasticsearch.distribution</groupId>
+        <artifactId>elasticsearch-distribution</artifactId>
+        <version>2.0.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>elasticsearch-distribution-tests</artifactId>
+    <name>Elasticsearch Distribution Tests</name>
+    <packaging>pom</packaging>
+
+    <!-- The documentation for how to run this is in ../Vagrantfile -->
+    <properties>
+      <testScripts>/vagrant/tests/src/test/resources/packaging/scripts/*.bats</testScripts>
+      <testCommand>sudo ES_CLEAN_BEFORE_TEST=true bats ${testScripts}</testCommand>
+
+      <allDebBoxes>precise, trusty, vivid, wheezy, jessie</allDebBoxes>
+      <allRpmBoxes>centos-6, centos-7, fedora-22</allRpmBoxes>
+
+      <debBoxes>trusty</debBoxes>
+      <rpmBoxes>centos-7.0</rpmBoxes>
+
+      <!-- Unless rpmbuild is available on the host we can't test rpm based
+           boxes because we can't build the rpm and they fail without the rpm.
+           So to get good coverage you'll need to run this on a system with
+           rpmbuild installed - either osx via homebrew or fedora/centos/rhel.
+           -->
+      <proposedBoxesToTest>${debBoxes}</proposedBoxesToTest>
+
+      <!-- redeclared here because you can't use properties from the parent's
+           pom during profile activation because maven hates everyone. -->
+      <!-- rpmbuild location : default to /usr/bin/rpmbuild -->
+      <packaging.rpm.rpmbuild>/usr/bin/rpmbuild</packaging.rpm.rpmbuild>
```

Comment:
this might fail with osx boxes that have rpmbuild installed (via brew), but it is unfortunately rpm5 and this creates packages that cannot be installed with rpm4... my centos-7 run seemed to be able to cope with this though